dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides

```diff
@@ -0,0 +1,58 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+
+WS-TraceGroup: WSSECURITY
+
+cxfVersion=2.6.2
+
+Bundle-Name: com.ibm.ws.org.apache.cxf.ws.security.2.6.2
+Bundle-Description: cxf ws security, version 2.6.2
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.ws.security.2.6.2
+Bundle-ActivationPolicy: lazy
+
+
+Export-Package: \
+  org.apache.cxf.ws.security.*
+
+Import-Package:  \
```

Comment:
Similar to the Export-Package, make sure the import statement matches exactly. I don't see a `*` import so be very careful to make sure you're importing every package that's needed by this bundle. If not, runtime behavior may be effected.